src/main/fc/runtime_config.c

```diff
@@ -15,10 +15,16 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// FIXME this file has started to resemble a garbage bin.  move everything out into appropriate locations.
+
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
```

Comment:
are theese files needed becuse of the include of io/beeper.h? - include it in the io/beeper.h instead